localization/english/wc_other_titles_l_english.yml

```diff
@@ -1,173 +1,674 @@
 ﻿l_english:
- k_atalai:0 "Atal'ai"
- 
- d_church_of_the_holy_light:0 "Church of the Holy Light"
- d_church_of_the_holy_light_article:0 "the "
- 
- d_church_scarlet_light:0 "Scarlet Monastery"
- d_church_scarlet_light_article:0 "the "
- 
- d_naaru:0 "Faithful of Naaru"
- d_hand_of_argus:0 "Hand of Argus"
- d_bloodscalp:0 "Bloodscalp"
- d_skullsplitter:0 "Skullsplitter"
- 
+ #########################
+ #		Without land		   #
```

Comment:
Again, do no sort the localization files. It's said in [Our Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/Localization-Guidelines).

You can move religious and holy order titles to different files. It's fine, it's allowed.

Reply:
done in 31e532701